src/providers/export/tensorFlowRecords.ts

```diff
@@ -256,6 +155,14 @@ export class TFRecordsJsonExportProvider extends ExportProvider<ITFRecordsJsonEx
         });
     }
 
+    private async writeTFRecords(fileNamePath: string, buffers: Buffer[]) {
```

Comment:
Looks like you set yourself up well to support the feature where we can combine output into a single tfrecords file.